src/platforms/javascript/common/install/index.mdx

```diff
@@ -20,3 +20,35 @@ notSupported:
 ---
 
 <PageGrid />
+
+## When to use which Installation Method
+
+Depending on the concrete needs of your application, you may wonder which installation method you should use.
+
+### Why to use the Loader Script over NPM
```

Comment:
```suggestion
### Reasons To Choose The Loader Script
```

Just a suggestion: Could also be "Why To Choose..." I think.

Reply:
I like the suggestion above: "### Reasons To Choose The Loader Script"

If we don't accept the above suggestion, then I'd say "Why to use..." sounds funny. IMO it should be either "Why use..." or "When to use..."

Reply:
```suggestion
### When to use the Loader Script over NPM
```

Reply:
"Why choose...over" makes more sense to me than "When to use...over"